mlx_audio/tts/generate.py

```diff
@@ -117,5 +117,59 @@ def main():
         traceback.print_exc()
 
 
+def generate_audio(text: str,
```

Comment:
This is a great handy function!

But I feel like it leaves duplicate code with the one above.

I would recommend merging them into one.

Reply:
No problem! I didn't want to interfere with the existing code, but you're absolutely right - it's better to avoid duplication. I'll have an update soon!

Reply:
Done! I submitted commit that unifying the logic for both command-line and script-based execution. I also tested both use cases.

Reply:
Added from_cli flag: a simple way to handle CLI vs. script execution differences. This ensures that CLI-generated files get the _000 suffix as expected, while script-based calls save files without a suffix (e.g., audiobook_chapter1.wav).

Reply:
> Added from_cli flag: a simple way to handle CLI vs. script execution differences. This ensures that CLI-generated files get the _000 suffix as expected, while script-based calls save files without a suffix (e.g., audiobook_chapter1.wav).

You need the suffix for each chunk otherwise you final audio will be overriden witht the last audio.

If you want to generate in chapters then I suggest using --join-audio flag, this takes all the audio chunks and creates one audio file called for example `audiobook_chapter1.wav`.

Then you just iterate chapter-by-chapter.